website/docs/r/datasource_google_iam_role.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: "google"
+page_title: "Google: google_iam_role"
+sidebar_current: "docs-google-datasource-folder"
+description: |-
+  Get information about a Google IAM Role.
+---
+
+# google\_iam\_role
+
+Use this data source to get information about a Google IAM Role.
+
+```hcl
+data "google_iam_role" "roleinfo" {
+  name = "roles/compute.viewer"
+}
+
+output "the_role_permissions" {
+  value = "${data.google_iam_role.roleinfo.included_permissions}"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` (Required) - The name of the Role to lookup.
```

Comment:
Can you include the part about the format from here? https://cloud.google.com/iam/reference/rest/v1/roles/get